app/src/test/java/com/account/test/AddRandomSuffixToUsernameServiceTest.java

```diff
@@ -0,0 +1,23 @@
+package com.account.test;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.account.service.AddRandomSuffixToUsernameService;
+
+@SpringBootTest
```

Comment:
In this test, u should use Mockito https://www.baeldung.com/mockito-annotations
Test only service in isolation with other systems.